players/g1_player.py

```diff
@@ -455,12 +460,13 @@ def play(self, score: int, golf_map: sympy.Polygon, target: sympy.geometry.Point
             v = np.array(target_point) - current_point
             # Unit vector pointing from current to target
             u = v / original_dist
-            if original_dist >= 20.0:
+            if original_dist >= 20.0 or (original_dist < 20 and is_in_sand_trap(
```

Comment:
Nice, taking into account of the fact that shots from sand trap are air-borne. Since we are aiming at the goal, which we kind of assumed that it would on green, so it would always roll.

Questions:

1. is it possible for the goal to be in a sand trap?
2. is `origin_dist < 20` necessary here? The or branch would be tested iff `original_dist < 20`

Reply:
no the goal cannot be in sand traps